src/testing/tests.cpp

```diff
@@ -169,6 +170,8 @@ ::select_test(const AllParam *const parameters_input,
         if constexpr (dim==3 && nstate==dim+2) return std::make_unique<TaylorGreenVortexEnergyCheck<dim,nstate>>(parameters_input,parameter_handler_input);
     } else if(test_type == Test_enum::taylor_green_vortex_restart_check) {
         if constexpr (dim==3 && nstate==dim+2) return std::make_unique<TaylorGreenVortexRestartCheck<dim,nstate>>(parameters_input,parameter_handler_input);
+    } else if(test_type == Test_enum::time_refinement_study_advection) {
```

Comment:
rename to `time_refinement_study` to be consistent with the class name since the advection part is handled by the `.prm` file and denoted in the ctest name

Reply:
Renaming to `time_refinement_study` in next commit.